src/_handler.ts

```diff
@@ -1,13 +1,120 @@
-import express from 'express';
+import path from 'path';
+import { tmpdir } from 'os';
+import { createHash } from 'crypto';
+import through from 'through2';
+import express, { Request } from 'express';
+import { createReadStream, readFile, mkdtemp } from 'mz/fs';
```

Comment:
In `node12` runtime possible `import { promises as fs } from 'fs';`

Reply:
Ack